pom.xml

```diff
@@ -2521,6 +2528,13 @@
               <javacArg>${java.version}</javacArg>
               <javacArg>-Xlint:all,-serial,-path,-try</javacArg>
             </javacArgs>
+            <compilerPlugins>
```

Comment:
Scala 2.12 need this plugin but Scala 2.13 not,  haven't found out how to not configure the plug-in and the related arg `-P:silencer:globalFilters=xxx` in Scala 2.13 profile

Reply:
Need found a way to remove  configure of `silencer-plugin` in Scala 2.13 profile else sbt build will failed 
![image](https://user-images.githubusercontent.com/1475305/102471925-bae57400-4090-11eb-900d-5e80eed20c5c.png)